### PR TITLE
Feature/hawk2

### PR DESCRIPTION
**Pull request by taroth21**
@krig: Please review and provide feedback (preferably by adding comments to the respective lines here on GH). 
All in all, 8 questions to you are included in the following XML sources:
- xml/ha_config_hawk2.xml
- xml/ha_hawk2_cons_i.xml
- xml/ha_hawk2_manage_i.xml 
- xml/ha_hawk2_rsc_i.xml

Just search for 'krig' to find them. Tack! :)

### CODE REVIEW

File: xml/ha_config_hawk2.xml [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+   <para> By default, users logged in as &rootuser; or <systemitem
+     class="username">hacluster</systemitem> have full read-write access to all
+    cluster configuration tasks. However, <xref linkend="cha.ha.acl"
+     xrefstyle="select:title"/> (ACLs) can be used to define fine-grained access
+    permissions. </para>
+   <para> If ACLs are enabled in the CRM, the available functions in &hawk;2
+    depend on the user role and access permissions assigned to you. In addition,
+    the following functions in &hawk;2 can only be executed by the user
+     <systemitem class="username">hacluster</systemitem>: <remark>remark
+     2015-11-24: krig, is the following still valid?</remark></para>
```
Comment (krig) [inline xml/ha_config_hawk2.xml]: Yes, this is still the case.

File: xml/ha_hawk2_manage_i.xml [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+    <title>Do Not Touch Services Managed by the Cluster</title>
+    <para> When managing a resource via the &hasi;, the same resource must not be started or
+     stopped otherwise (outside of the cluster, for example manually or on boot or reboot). The
+     &hasi; software is responsible for all service start or stop actions. </para>
+    <para> However, if you want to check if the service is configured properly, start it manually,
+     but make sure that it is stopped again before &ha; takes over. </para>
+    <para> For interventions in resources that are currently managed by the cluster, set the
+     resource to <literal>maintenance mode</literal> first. <remark>taroth 2015-12-04: krig - how
+      does that work with hawk2? does it work already? I searched in the Operations column for a
+      resource but could not find anything there...</remark>
```
Comment (krig) [inline xml/ha_hawk2_manage_i.xml]: Ah! That's the "Unmanage" operation on resources.

File: xml/ha_hawk2_rsc_i.xml [hunk 3]
```diff
@@ -0,0 +1,755 @@
...
+      Enter a unique <guimenu>Group ID</guimenu>.
+     </para>
+    </step>
+    <step>
+     <para>
+      To define the group members, select one or multiple entries in the list
+      of <guimenu>Children</guimenu>. <remark>taroth 2015-11-30: krig: how does resorting group
+       members work in hawk2? I did not find a way to change the sorting order...<!-- To define the order
+      of the group members, you currently need to add and remove them in the
+      order you desire.--></remark>
```
Comment (krig) [inline xml/ha_hawk2_rsc_i.xml]: So, caveat: This only works with the maintenance update as the UI was buggy and always re-sorted the resources in the GM version: Resort group members by dragging and dropping them into the order you want. Drag and drop using the "handle" on the right.

File: xml/ha_hawk2_manage_i.xml [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
+       <guimenu>Resources</guimenu> also shows the <guimenu>Status</guimenu>. </para>
+    </step>
+    <step>
+     <para> Go to the resource to clean up. In the <guimenu>Operations</guimenu> column click the
+      arrow down button and select <guimenu>Cleanup</guimenu>. To continue, confirm the message that
+      appears. </para>
+     <para>
+      <remark>taroth 2015-12-07: krig, is the following still true?</remark>This executes the
+      command <command>crm resource cleanup</command> for the specified resource on the specified
+      node. </para>
```
Comment (krig) [inline xml/ha_hawk2_manage_i.xml]: Not quite, clean up cleans the resource on all nodes, not the specified node. But this was also the case in Hawk 1...

File: xml/ha_hawk2_manage_i.xml [hunk 5]
```diff
@@ -0,0 +1,365 @@
...
+    </step>
+    <step>
+     <para> In the left navigation bar, select <guimenu>Status</guimenu>. The list of
+       <guimenu>Resources</guimenu> also shows the <guimenu>Status</guimenu>. </para>
+    </step>
+    <step>
+     <para>In the list of <guimenu>Resources</guimenu>, go to the respective resource.</para>
+    </step>
+    <step>
+     <para><remark>taroth 2015-12-07: krig, is there any way to select which node to migrate the
```
Comment (krig) [inline xml/ha_hawk2_manage_i.xml]: No, not currently, but do file a feature request for that, would be useful

File: xml/ha_hawk2_manage_i.xml [hunk 6]
```diff
@@ -0,0 +1,365 @@
...
+       resource to? krig: also, Hawk1 has two options: "Move" (lets you select where to move the rsc
+       to and creates a location constraint with an INFINITY score for the destination node) and
+       "Away from current node" (creates a location constraint with a -INFINITY score for the
+       current node) - si there anything like this with Hawk2? </remark>In the
```
Comment (krig) [inline xml/ha_hawk2_manage_i.xml]: Aha, so we have lost "Move" and currently only have "Away from current node". Maybe file a bug for this, actually.

File: xml/ha_hawk2_manage_i.xml [hunk 7]
```diff
@@ -0,0 +1,365 @@
...
+    </step>
+    <step>
+     <para>In the list of <guimenu>Resources</guimenu>, go to the respective resource.</para>
+    </step>
+    <step>
+     <para>In the <guimenu>Operations</guimenu> column click the arrow down button and select
+       <guimenu>Unmigrate</guimenu>. To continue, confirm the message that appears.</para>
+     <para>
+      <remark>taroth 2015-12-07: krig, is the following still correct? and which crm shell command
+       is used? the "Command Log" unfortunately did not show anything...</remark> The resource
```
Comment (krig) [inline xml/ha_hawk2_manage_i.xml]: Yes, still correct. These commands don't actually call `crm` but use the pacemaker `crm_resource` command directly, but yes, we should probably log them in the command log too.

File: xml/ha_hawk2_rsc_i.xml [hunk 8]
```diff
@@ -0,0 +1,755 @@
...
+    </step>
+   </procedure>
+
+   <para> For the processes which take place if the resource monitor detects a failure, refer to
+     <xref linkend="sec.ha.config.basics.monitoring"/>. </para>
+   <para>
+    <remark>taroth 2015-12-04: krig - is the following still true?</remark> To view resource
+    failures, switch to the <guimenu>Status</guimenu> screen in &hawk; and select the resource
+    you are interested in. Click the magnifier icon in the <guimenu>Operations</guimenu> column to
+    view the resources <guimenu>Details</guimenu>. </para>
```
Comment (krig) [inline xml/ha_hawk2_rsc_i.xml]: Yes, but maybe add that there is also a "Recent Events" dialog now which shows a list of recent actions performed for a given resource. Failures are marked as red in that list.

File: xml/ha_hawk2_manage_i.xml (hunk 6 above)
Comment (taroth21) [inline xml/ha_hawk2_manage_i.xml]: Done, filed https://bugzilla.suse.com/show_bug.cgi?id=958386 (this also covers your note right above this one).

File: xml/ha_hawk2_manage_i.xml (hunk 5 above)
Comment (taroth21) [inline xml/ha_hawk2_manage_i.xml]: See next comment.

File: xml/ha_config_hawk2.xml (hunk 1 above)
Comment (taroth21) [inline xml/ha_config_hawk2.xml]: Thanks, changed accordingly.

File: xml/ha_hawk2_manage_i.xml (hunk 2 above)
Comment (taroth21) [inline xml/ha_hawk2_manage_i.xml]: Hm, sure? I thought 'unmanaged' and 'maintenance mode' are two different things... see https://www.suse.com/documentation/sle-ha-12/book_sleha/data/sec_ha_config_basics_resources.html#sec_ha_config_basics_meta_attr

File: xml/ha_hawk2_manage_i.xml (hunk 4 above)
Comment (taroth21) [inline xml/ha_hawk2_manage_i.xml]: Ok, changed accordingly.

File: xml/ha_hawk2_manage_i.xml (hunk 7 above)
Comment (taroth21) [inline xml/ha_hawk2_manage_i.xml]: Thanks for confirming. Added that it calls crm_resource -U. And yes, adding also crm_* commands to the command log if they are executed would be useful IMHO.

File: xml/ha_hawk2_rsc_i.xml (hunk 3 above)
Comment (taroth21) [inline xml/ha_hawk2_rsc_i.xml]: Thanks, added that.

File: xml/ha_hawk2_rsc_i.xml (hunk 8 above)
Comment (taroth21) [inline xml/ha_hawk2_rsc_i.xml]: Thanks, added that.

**Pull request merged**